gradebookng/api/src/main/java/org/sakaiproject/grading/api/model/GradebookAssignment.java

```diff
@@ -542,11 +542,11 @@ public String getItemType() {
                 return this.itemType;
             }
 
-            if (gb.getGradeType() == GradeType.POINTS) {
+            if (GradeType.POINTS.equals(gb.getGradeType())) {
```

Comment:
does it make sense to set a default to points?

Reply:
Gradebook has the default set:
private GradeType gradeType = GradeType.POINTS;

Reply:
awesome. thanks!